### PR TITLE
PLATFORM-3888

### PR DESCRIPTION
**Pull request by harnash**

### CODE REVIEW

File: resizer.go [hunk 1]
```diff
@@ -61,8 +61,10 @@ func resizer(buf []byte, o Options) ([]byte, error) {
 			factor = 1.0
 			shrink = 1
 			residual = 0
-			o.Width = inWidth
-			o.Height = inHeight
+			//if !o.Embed {
```
Comment (harnash) [inline resizer.go]: Is this needed?

File: resizer.go [hunk 2]
```diff
@@ -567,3 +607,106 @@ func getAngle(angle Angle) Angle {
...
+			if err != nil {
+				return nil, err
+			}
+		}
+	}
+
+	// now we have the cropped image of o.AreaWidth, o.AreaHeight size; resize it to match the target size
+	inWidth := int(image.Xsize)
+	inHeight := int(image.Ysize)
+	fmt.Printf("Cropped the image, the size is (%v, %v), output size should be (%v, %v)\n",
```
Comment (harnash) [inline resizer.go]: Nope :) and other instances in this file

**Review by harnash — COMMENTED** [review]
I would really like some tests for the new functionality

File: resizer.go (hunk 1 above)
Comment (themech) [reply resizer.go]: removed

File: resizer.go (hunk 2 above)
Comment (themech) [reply resizer.go]: I know, remove all the printfs :)

**Review by themech — APPROVED** [review]

**Review by themech — APPROVED** [review]

**Pull request merged**